apps/microsoft-teams/app-actions/src/helpers/entry-activity.ts

```diff
@@ -0,0 +1,127 @@
+import { LocaleProps, EntryProps, PlainClientAPI, SysLink } from 'contentful-management';
+import { Action, ActionType, EntryActivity, EntryEvent, Topic } from '../types';
+import { TOPIC_ACTION_MAP } from '../constants';
+
+// comment
+export const buildEntryActivity = async (
+  entryEvent: EntryEvent,
+  cma: PlainClientAPI
+): Promise<EntryActivity> => {
+  const { entry, topic, eventDatetime } = entryEvent;
+
+  const entryId = entry.sys.id;
+  const spaceId = entry.sys.space.sys.id;
+  const contentTypeId = entry.sys.contentType.sys.id;
+
+  const { name: contentTypeName, displayField } = await cma.contentType.get({ contentTypeId });
+  const localeCollection = await cma.locale.getMany({});
+
+  const entryTitle = computeEntryTitle(entry, displayField, localeCollection.items);
+
+  // we can't get the space name for now
+  const spaceName = 'TODO: Space name';
+
+  const action = topicToAction(topic);
+  const actorName = await computeActorName(entry, action, cma);
```

Comment:
Eventually remove this since we won't be including it

Reply:
Added a sub task 👍